config/jobs/kubernetes/sig-instrumentation/sig-instrumentation-kind-periodics.yaml

```diff
@@ -31,19 +33,24 @@ periodics:
       - wrapper.sh
       - bash
       - -c
-      - curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" && e2e-k8s.sh
+      # We need the modified e2e-k8s.sh from https://github.com/kubernetes-sigs/kind/pull/3023
+      - curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" && curl -sSL https://github.com/pohly/kind/raw/e2e-feature-gates/hack/ci/e2e-k8s.sh >e2e-k8s.sh && chmod u+x e2e-k8s.sh && e2e-k8s.sh
```

Comment:
Not sure I understand what's the goal of this change. Maybe you can work with @BenTheElder to avoid patching.

Reply:
I need some way of controlling feature gates and the runtime config. See also https://kubernetes.slack.com/archives/CEKK1KTN2/p1671006561936529

Reply:
The current e2e-k8s.sh doesn't support changing those.